web-console/1-homepage/catalog.md

```diff
@@ -1,51 +1,47 @@
-*******************************
-Things you may want to include in this file
-Browse
-filter
+# Catalog
 
-*******************************
+- The Catalog section is found at the bottom of the web console homepage.
+- The Catalog contains provisionable items and provides the means to browse, sort, and filter them.
 
 
-# Template
+## Browse Catalog
 
-- This is a description of the section and where it is located/how to get to it
-- This is a description of what this section is for
+![catalog](img/catalog-01.png)
+- Initially, upon arriving on the page, the `All Categories` section is expanded and the resources are sorted alphabetically by name.
```

Comment:
@cshinn do you think it makes more sense to just say "All" at both levels of categorization?